ArticleTemplates/assets/css/style.css

```diff
@@ -845,14 +862,39 @@ Display which keywords an article has been tagged with.
     .tags {
       font-size: 1.6rem;
       line-height: 2.2rem; } }
-  @media all and (max-width: 28.115em) {
-    .tags .inline-list__item {
-      margin-bottom: 4px; } }
-  .tags .inline-list__item:first-of-type {
-    font-weight: bold; }
-  .tags .inline-list__item:last-child .tags__separator {
-    display: none;
-    visibility: hidden; }
+  .tags .hide-tags {
+    display: none; }
+  .tags .show-tags {
+    display: inline-block; }
+  .tags .inline-list__item {
+    margin: 1rem 0;
+    /*        @include mq($to: col1) {
```

Comment:
it can be a good idea to avoid commenting out code, as it's available in the git history anyway.  I used to do it all the time but I was soon put right ;)